### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by DavidjohnBlodgett**

### CODE REVIEW

**Comment by DavidjohnBlodgett** [comment]
@heckj @benbp @rolandpoulter @tldavies

**Comment by DavidjohnBlodgett** [comment]
@heckj @benbp @rolandpoulter @tldavies ok, ok... this is a little cleaner!

**Comment by jlongever** [comment]
+1

File: lib/app.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ di.annotate(Runner, new di.Inject(
 );
 
 function Runner(core, server, messenger, Promise) {
+    var server = server.create();
```
Comment (heckj) [inline lib/app.js]: you'll want to deal with this one - you're redefining "server" here as you're creating it. not sure if the `server` getting passed in is an object or Constructor - if Constructor, then just capitalize the one getting passed in and update the method appropriately.

**Comment by heckj** [comment]
The semi-colons and redefining `server` should get sorted, but otherwise +1

File: lib/app.js (hunk 1 above)
Comment (rolandpoulter) [inline lib/app.js]: You might want to rename this variable since it is already used. Or delete the `var` statement.

**Comment by rolandpoulter** [comment]
I agree with Joe. +1 once those lint errors are fixed.

**Comment by DavidjohnBlodgett** [comment]
@heckj @benbp @rolandpoulter @tldavies everything looks green :dancer:

**Comment by heckj** [comment]
:+1:

**Comment by rolandpoulter** [comment]
:+1:

**Pull request merged**